src/hardware/hantek-6xxx/api.c

```diff
@@ -794,15 +813,25 @@ static int dev_acquisition_start(const struct sr_dev_inst *sdi, void *cb_data)
 	std_session_send_df_header(cb_data, LOG_PREFIX);
 
 	devc->samp_received = 0;
+#if FLUSH_PACKET_SIZE > 0
 	devc->dev_state = FLUSH;
-
+#else
+	devc->dev_state = CAPTURE;
+#endif
+	
```

Comment:
Ok, but I think we should remove flush totally. In theory we should get good samples directly.